api/Path2D.json

```diff
@@ -78,10 +78,10 @@
               "version_added": "24"
             },
             "safari": {
-              "version_added": "10"
+              "version_added": "7"
```

Comment:
This is the constructor and the "7" made me skeptical after a bunch of other suspicious "7" values today. But this seems right in manual testing, and the rest of the data makes sense in light of this.